experimental/FTheoryTools/test/literature_models.jl

```diff
@@ -799,28 +801,20 @@ foah16_B3_weier = literature_model(arxiv_id = "1408.4808", equation = "3.203", t
   @test length(singular_loci(foah14_B3_weier)) == 4
   @test length(singular_loci(foah15_B3_weier)) == 5
   @test length(singular_loci(foah16_B3_weier)) == 4
-  @test singular_loci(foah4_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah6_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah8_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah8_B3_weier)[3][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah9_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah10_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah10_B3_weier)[3][2:3] == ((0, 0, 3), "Split I_3")
-  @test singular_loci(foah11_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah11_B3_weier)[3][2:3] == ((0, 0, 3), "Split I_3")
-  @test singular_loci(foah12_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah12_B3_weier)[3][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah13_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah13_B3_weier)[3][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah13_B3_weier)[4][2:3] == ((0, 0, 4), "Split I_4")
-  @test singular_loci(foah14_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah14_B3_weier)[3][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah14_B3_weier)[4][2:3] == ((0, 0, 3), "Split I_3")
-  @test singular_loci(foah15_B3_weier)[2][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah15_B3_weier)[3][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah15_B3_weier)[4][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah15_B3_weier)[5][2:3] == ((0, 0, 2), "Non-split I_2")
-  @test singular_loci(foah16_B3_weier)[2][2:3] == ((0, 0, 3), "Split I_3")
-  @test singular_loci(foah16_B3_weier)[3][2:3] == ((0, 0, 3), "Split I_3")
-  @test singular_loci(foah16_B3_weier)[4][2:3] == ((0, 0, 3), "Split I_3")
+  @test sort([k[2:3] for k in singular_loci(foah1_B3_weier)]) == [((0, 0, 1), "I_1")]
```

Comment:
```suggestion
  @test issetequal([k[2:3] for k in singular_loci(foah1_B3_weier)], [((0, 0, 1), "I_1")])
```
this is better than sorting

Reply:
This was of course meant to be applied for all of these cases :)

Reply:
Indeed. Working on this and the failed doctests, which need similar changes. Sadly, many instances that need changing...